CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+#
+# CMAKE SETUP (version, build style)
+#
+cmake_minimum_required (VERSION 2.8 FATAL_ERROR)
+project ("Quick-DER" C)
+
+set (CMAKE_MACOSX_RPATH 0)  # Don't use rpaths (but we don't build executables anyway)
```

Comment:
RPATH kan ook in libraries geset worden, die laden dan hun afhankelijkheden bij voorkeur uit dat pad.  Bigtime op NixOS en, zo leerde ik hierdoor, blijkbaar ook op Mac OS X, waar rpath default actief zou zijn.